deploy/dependencies/korrel8r/serviceaccount.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: korrel8r-sa
```

should this be:
```suggestion
  name: obo-korrel8r-sa
```
based on the change on the deployment?

---

Agree with that, also note that changing this would also change the ServiceAccountName later on

---

From what I understand The kustomization file updates the sa and all other resources with a prefix "obo" https://github.com/rhobs/observability-operator/blob/main/deploy/dependencies/kustomization.yaml#L43. So when I run the `make bundle-image`  the new manifests created already have the `obo` prefix automatically. Is there another way this is commonly built where the prefix  does not get added automatically?

---

Oh, yeah, forgot about that part tbh! disregard this then ;) @jan--f mind taking a look as well? Thanks!